Rakefile

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rspec/core/rake_task'
+RSpec::Core::RakeTask.new(:spec)
+
+require_relative 'tasks/migration_generator'
```

Comment:
if this starts to bloat, there's some ruby magic we could use to require everything in the subdirectory, like:

```
Dir[File.expand_path('tasks/*.rb', File.dirname(__FILE__))].each do |file|
  require file
end
```